lib/main.dart

```diff
@@ -42,14 +42,22 @@ class MyHomePage extends StatefulWidget {
 class _MyHomePageState extends State<MyHomePage> {
   Game _game = _createGame(initialGridName);
   final ValueNotifier<String> _grid = ValueNotifier<String>(initialGridName);
+  late ValueNotifier<Coordinates> _lastMove;
 
```

Comment:
Remove late and assign ValueNotifier<Coordinates>(null) right here, in initializer.